markdown/events/2023/hackathon-march-2023/index.md

```diff
@@ -246,23 +252,56 @@ However, if you can't make it, these will be recorded and can be viewed afterwar
     </table>
 </div>
 
-_Please note that this is a preliminary schedule that may be subject to change._
+_Reminder: Clocks go forward one hour in Europe on March 26, 2023._
+
+# Pre-hackathon checklist
+
+Ensure you have read/signed up/joined/installed the following resources before the hackathon.
+
+- Check you agree with the [Code of Conduct](https://nf-co.re/code_of_conduct) of the event
+- If you haven’t already, set up a GitHub account and [join the nf-core GitHub organization](https://nf-co.re/join#github)
+- Join the nf-core slack and the [#hackathon-march-2023 channel](https://nfcore.slack.com/archives/C04U29Y97SB)
+- Create a Gather account using the email you used to register (see above)
+- Have [Nextflow](https://nextflow.io/), [nf-core/tools](https://nf-co.re/tools/#installation), and one of [Docker](https://docs.docker.com/get-docker/), Singularity, or [Conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html)/[Mamba](https://mamba.readthedocs.io/en/latest/installation.html) installed on your computer
+- Familiarize yourself with the [documentation on the nf-core website](https://nf-co.re/docs)
 
-## How we will work
+# How to hack!
 
 We will be a lot of people working in parallel during this hackathon, so to stay organized we have a recommended workflow:
 
 1. :speech_balloon: Chat with your group (on slack or gather town) to get an overview of what is going on.
-2. <i class="fab fa-slack"></i> Join the relevant Slack channel(s) to stay up to date and discuss with your project members (channels will be made available closer to the event).
-3. <i class="fab fa-github"></i> Find a task to work on using the GitHub Project Board (the board will be made available closer to the event).
+2. <i class="fab fa-slack"></i> Join the relevant Slack channel(s) to stay up to date and discuss with your project members.
+   - [Documentation channel](https://nfcore.slack.com/archives/C04UGQXUK2N)
+   - [Infrastructure channel](https://nfcore.slack.com/archives/C04V6FR8LN4)
+   - [Pipelines channel](https://nfcore.slack.com/archives/C04UK9FMKN0)
+   - [Modules channel](https://nfcore.slack.com/archives/C04UA7FFWES)
+   - [Subworkflows channel](https://nfcore.slack.com/archives/C04UGMMT3JP)
+3. <i class="fab fa-github"></i> Find a task to work on using the [GitHub Project Board](https://github.com/orgs/nf-core/projects/41/views/1).
```

Comment:
Hello! I see the link to the project board is not the right one :) 
```suggestion
3. <i class="fab fa-github"></i> Find a task to work on using the [GitHub Project Board](https://github.com/orgs/nf-core/projects/38/views/16).
```

Reply:
Thank you!